persepolis/scripts/persepolis.py

```diff
@@ -186,19 +186,21 @@ def setPersepolisColorScheme(self, color_scheme):
 # This dirty trick will show Persepolis version when there are unknown args
 # Unknown args are sent by Browsers for NHM
 if args.parent_window or unkownargs:
-
+   
     # Platform specific configuration
     if os_type == OS.WINDOWS:
         # Set the default I/O mode to O_BINARY in windows
         import msvcrt
         msvcrt.setmode(sys.stdin.fileno(), os.O_BINARY)
         msvcrt.setmode(sys.stdout.fileno(), os.O_BINARY)
-
-    # Send message to browsers plugin
-    message = '{"enable": true, "version": "1.85"}'.encode('utf-8')
-    sys.stdout.buffer.write((struct.pack('i', len(message))))
-    sys.stdout.buffer.write(message)
-    sys.stdout.flush()
+    else:
+        # TODO: fix windows error
```

Comment:
This line is in `else` part of  `if os_type == OS.WINDOWS:` so it shouldn't throw error for Windows !?

Reply:
Yes.   
It could be in another `if os_type != OS.WINDOWS:` but this version also does the work.

Reply:
Oh ok i misunderstood the comment.